tests/minimal_vf/config.toml

```diff
@@ -0,0 +1,20 @@
+output_file = "MinimalVF.ttf"
+
+[axis.wght]
+name = "Weight"
+# ufo2ft fails if default is not at a master.
+default = 300
```

Comment:
it's not ufo2ft's fault, it's the way VF works. One of the masters has to be the default. If you don't care which one just use the minimum. Making an arbitrary instance from an initial set of masters and re-inserting this instance as a new master and making it the new default creates a whole new design-space which may not necessarily give identical results as the original set of masters. So the user has to provide the full set of masters, including the default one which is the most important of all.

Reply:
I don't think the comment implies ufo2ft is doing anything wrong?

Reply:
It's fine, I misunderstood the "fails" ;)